variables.tf

```diff
@@ -267,3 +267,21 @@ variable "additional_databases" {
   }))
   default = []
 }
+
+variable "maintenance_window" {
+  description = <<-EOT
+  day: The day of week (1-7), starting on Monday.
+  hour: The hour of day (0-23), ignored if day not set.
+  update_track: The update track of maintenance window. Can be either `canary` or `stable`.
+  EOT
+  type = object({
+    day_utc      = number
+    hour_utc     = number
+    update_track = string
+  })
+  default = {
+    day_utc      = 1  # implies Tuesday in MYT
```

Comment:
Actually, this comment (when read by itself) is misleading. `day_utc = 1` does not imply Tuesday in MYT. Rather, it's more like `day_utc = 1 AND time_utc > 18` implies Tuesday MYT.

To illustrate what you tried to say here, you can write this full matter in the description itself at its end. So users of the module will know from the doc what is expected.